net/third_party/quiche/src/quiche/quic/core/quic_received_packet_manager.cc

```diff
@@ -69,6 +74,27 @@ void QuicReceivedPacketManager::SetFromConfig(const QuicConfig& config,
   if (config.HasClientSentConnectionOption(k1ACK, perspective)) {
     one_immediate_ack_ = true;
   }
+  if (config.HasClientSentConnectionOption(kAKD1, perspective)) {
```

Comment:
We'll want to use `defined(USE_COBALT_CUSTOMIZATIONS)` for the added changes as well

Reply:
Oh, I missed these. Good catch.